saq/queue/postgres.py

```diff
@@ -306,11 +305,42 @@ async def sweep(self, lock: int = 60, abort: float = 5.0) -> list[str]:
             cursor.nextset()
             cursor.nextset()
             results = await cursor.fetchall()
-            for result in results:
-                job = self.deserialize(result[0])
+            for key, job_bytes, objid in results:
+                job = self.deserialize(job_bytes)
                 if not job:
```

Comment:
how can job be null?

Reply:
It's theoretically possible the job column could be null, but there's no code path to that situation. Maybe if someone logs into the database and updates the job table manually. Want me to remove this? And/or add a not null constraint on the job column?

Reply:
yea sure, let's make job not null